### DISCUSSION TITLE
How do I use Kingfisher in a swift playground in Xcode 13?

### DISCUSSION THREAD

**Discussion by S0ulf3re** [onevcat/Kingfisher#1833] [Q&A]
I'm trying to use kingfisher within an Xcode playground. So I downloaded it and dragged the file into sources and resources, but when I try to import it using ``` import kingfisher ```, I get a module not found error. What am I doing wrong?

**Comment by onevcat** [comment]
It does not support to be used in Playground yet due to some namespace conflicting. But this can be a very minor fix and now it supports there. I will submit a commit for it soon.

![截屏2021-10-13 15 26 12](https://user-images.githubusercontent.com/1019875/137078759-bd99d0af-8c24-48db-a851-96b5eb38b1cc.png)
